01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,14 @@
+for x in 1..20
+	if x% (3 * 5)==0
```

Comment:
```suggestion
	if x % (3 * 5) == 0
```

のように演算子の前後にはスペース1つを入れることが多いです。
他の部分も見直してみて下さい。

また、インデントはタブ文字ではなく半角スペース2つにしてください。タブキーで自動的にスペース2文字が入力されるよう、エディタの設定を変更しておくことをお勧めします。